Sources/PhoenixLiveViewNative/Views/PhxModernNavigationLink.swift

```diff
@@ -68,10 +68,12 @@ struct PhxModernNavigationLink<R: CustomRegistry>: View {
             
         case .push:
             func doPushNavigation() {
-                navCoordinator.sourceRect = source?.frameProvider() ?? .zero
-                navCoordinator.sourceElement = source?.element
-                navCoordinator.overrideOverlayView = overrideView?.view
-                navCoordinator.navigationPath.append(dest)
+                DispatchQueue.main.async {
+                    navCoordinator.sourceRect = source?.frameProvider() ?? .zero
+                    navCoordinator.sourceElement = source?.element
+                    navCoordinator.overrideOverlayView = overrideView?.view
+                    navCoordinator.navigationPath.append(dest)
+                }
```

Comment:
This change is unnecessary. The warning from SwiftUI that you probably saw about changes being made during a view update is spurious (see the comment several lines below this).